secure-add-selinux (#663)
centreon/centreon-documentation: en/administration/secure-platform.md

restricting specific applications to only viewing log files, while allowing other applications to append new data to
the log files.

SELinux implements Mandatory Access Control (MAC). Every process and system resource has a special security label
called an SELinux context. A SELinux context, sometimes referred to as an SELinux label, is an identifier which
abstracts away the system-level details and focuses on the security properties of the entity. Not only does this provides
a consistent way of referencing objects in the SELinux policy, but it also removes any ambiguity that can be found in
other identification methods. For example, a file can have multiple valid path names on a system that makes use of bind
mounts.

The SELinux policy uses these contexts in a series of rules which define how processes can interact with each other and
the various system resources. By default, the policy does not allow any interaction unless a rule explicitly grants access.

For more information about SELinux please see [Red Hat documentation](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/using_selinux/getting-started-with-selinux_using-selinux)

> ### Activer SELinux en mode permissif

Reviewer: ```suggestion
### Activate SELinux in permissive mode
```

# SELINUX= can take one of these three values:
#     enforcing - SELinux security policy is enforced.
#     permissive - SELinux prints warnings instead of enforcing.
#     disabled - No SELinux policy is loaded.
SELINUX=permissive
# SELINUXTYPE= can take one of three two values:
#     targeted - Targeted processes are protected,
#     minimum - Modification of targeted policy. Only selected processes are protected.
#     mls - Multi Level Security protection.
SELINUXTYPE=targeted
```

Then reboot your server:
```shell
shutdown -r now
```

### Install Centreon SELinux packages

Depending on the type of server, install the packages with the following command:

<!--DOCUSAURUS_CODE_TABS-->
<!--Central / Remote Server-->
   ```shell
   yum install centreon-common-selinux \
   centreon-web-selinux \
   centreon-broker-selinux \
   centreon-engine-selinux \
   centreon-gorgoned-selinux \
   centreon-plugins-selinux
   ```
<!--Poller-->
   ```shell
   yum install centreon-common-selinux \
   centreon-broker-selinux \
   centreon-engine-selinux \
   centreon-gorgoned-selinux \
   centreon-plugins-selinux
   ```
<!--Map server-->
   ```shell

>    yum install centreon-map-selinux

Reviewer: Isn't there a dependency between map and centreon-common?

Author: I didn't think so

   ```
<!--MBI server-->
   ```shell
   yum install centreon-mbi-selinux
   ```
<!--END_DOCUSAURUS_CODE_TABS-->

To check the installation, execute the following command:

```shell
semodule -l | grep centreon
```

Depending on your type of server, you can see:
```shell
centreon-broker	0.0.5
centreon-common	0.0.10
centreon-engine	0.0.8
centreon-gorgoned	0.0.3
centreon-plugins	0.0.2
centreon-web	0.0.8
```

### Audit logs and enable SELinux

> Before to enable SELinux in **enforcing** mode, you need to be sure that no errors appear using the following command:

Reviewer: ```suggestion
Before enabling SELinux in **enforcing** mode, you need to be sure that no errors appear using the following command:
```

components by the operating system.

> These rules are currently in **beta mode** and can be activated.
> You can activate them by following this procedure. In you detect of a problem,
> it is possible to disable SELinux globally and to send us your feedback in
> order to improve our rules on [Github](https://github.com/centreon/centreon).

### SELinux Overview

Security Enhanced Linux (SELinux) provides an additional layer of system security. SELinux fundamentally answers the
question: `May <subject> do <action> to <object>?`, for example: May a web server access files in users' home
directories?

The standard access policy based on the user, group, and other permissions, known as Discretionary Access Control
(DAC), does not enable system administrators to create comprehensive and fine-grained security policies, such as
restricting specific applications to only viewing log files, while allowing other applications to append new data to
the log files.

SELinux implements Mandatory Access Control (MAC). Every process and system resource has a special security label
called an SELinux context. A SELinux context, sometimes referred to as an SELinux label, is an identifier which

> abstractsaway the system-level details and focuses on the security properties of the entity. Not only does this provide

Reviewer: ```suggestion
abstracts away the system-level details and focuses on the security properties of the entity. Not only does this provides
```

a consistent way of referencing objects in the SELinux policy, but it also removes any ambiguity that can be found in
other identification methods. For example, a file can have multiple valid path names on a system that makes use of bind
mounts.

La politique SELinux utilise ces contextes dans une série de règles qui définissent comment les processus peuvent
interagir entre eux et avec les différentes ressources système. Par défaut, la stratégie n'autorise aucune interaction

> à moins qu'une règle n'accorde explicitement l'accès.

Reviewer: French part in english documentation
